CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - gRPC: accept dialer options in gRPC transportSpec config
+- Building yarpc against go1.14 and go1.15
```

Comment:
IMO there is no need of putting this in the changelog. Changelog should only contains entries which affect APIs of the library